src/map/script.cpp

```diff
@@ -26957,6 +26957,37 @@ BUILDIN_FUNC(has_autoloot) {
 	return SCRIPT_CMD_SUCCESS;
 }
 
+// ===================================
+// *autoloot(<rate>{, <char_id>});
+// This command sets the rate of autoloot (10000 = 100%).
+// Returns true on success and false on failure.
+// Note: It will toggle existing autoloot settings if rate value aren't provided.
+// ===================================
+BUILDIN_FUNC(autoloot) {
+	map_session_data *sd;
+
+	if (!script_charid2sd(3, sd)) {
+		script_pushint(st, false);
+		return SCRIPT_CMD_FAILURE;
+	}
+
+	int rate = 0;
```

Comment:
```suggestion
	int rate = 10000;
```

Reply:
i think this way it wont be able to toggle to off if user already set autoloot before this script command executed.